src/robotjs.cc

```diff
@@ -156,7 +156,7 @@ NAN_METHOD(mouseClick)
 
 	if (info.Length() == 2)
 	{
-		doubleC = info[1]->BooleanValue();
+		doubleC = info[1]->BooleanValue(Nan::GetCurrentContext()).FromJust();
```

Comment:
This line does not work for Electron v6 (NODE_MODULE_VERSION 73). I changed it to this and worked:

```c
		doubleC = Nan::To<bool>(info[1]).FromJust();
```

Reply:
Thank you @gera2ld